addons/Math/src/presenter.js

```diff
@@ -807,5 +812,60 @@ function AddonMath_create() {
         }
     }
 
+    presenter.printableController = null;
+    presenter.setPrintableController = function(controller) {
+        presenter.printableController = controller;
+    }
+
+    presenter.onBeforePrint = function(model) {
+        if (!presenter.printableController) return;
+        var configuration = presenter.convertModel(model);
+        var result = presenter.evaluateAllExpressions(configuration.expressions, configuration.variables, configuration.separators, true);
+        if (!result.isError) {
```

Comment:
Proponuję 
```
if (result.isError) {
 return;
}
```
Na ten moment jeżeli result.isError === true to będzie błąd, gdyż pętla z 831 lini korzysta ze słownika, który nie powstał.